utilities/src/biginteger/macros.rs

```diff
@@ -190,6 +190,28 @@ macro_rules! bigint_impl {
                 res
             }
 
+            #[inline]
+            fn to_bytes_be(&self) -> Vec<u8> {
+                let mut le_bytes = self.to_bytes_le();
+                le_bytes.reverse();
+                le_bytes
+            }
+
+            #[inline]
+            fn to_bytes_le(&self) -> Vec<u8> {
+                let array_map = self.0.iter().map(|limb| limb.to_le_bytes());
+                let mut res = Vec::<u8>::with_capacity($num_limbs * 8);
+                for limb in array_map {
+                    res.extend_from_slice(&limb);
+                }
+                res
+            }
+
+            #[inline]
+            fn to_biguint(&self) -> num_bigint::BigUint {
+                BigUint::from_bytes_le(&self.to_bytes_le())
+            }
+
```

Comment:
This is great! Let's keep these methods along with the impl in PR #245

Reply:
(I copied them from arkworks-rs, which we added the `to_biguint` just recently because otherwise printing out field elements is hard.)